GameData/RealismOverhaul/Engine_Configs/RD108_RD118_Config.cfg

```diff
@@ -776,7 +775,7 @@
 		ignitionReliabilityEnd = 0.981818
 		cycleReliabilityStart = 0.909091
 		cycleReliabilityEnd = 0.981818
-		techTransfer = RD-200:25&RD-108-8D75:50
```

Comment:
Should mention in the PR that it also fixes a bunch of bad tech transfer configs on other engines. Ideally these would be a separate PR though.